### PR TITLE
🐛 Source Github: add new stream `ProjectCards`

### LINKED ISSUE #10749: 🐛 Source Github: add new stream `ProjectCards`
## Tell us about the problem you're trying to solve
In scope of this [EPIC] #10280
Add new streams `ProjectCards`

## Describe the solution you’d like

### PR DESCRIPTION
**Pull request by grubberr**
## What
Add new streams `ProjectCards`

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [x] Build is successful
- [x] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [x] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [x] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [x] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [x] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-github/source_github/streams.py [hunk 1]
```diff
@@ -886,7 +886,7 @@ def stream_slices(
                 sync_mode=SyncMode.full_refresh, cursor_field=cursor_field, stream_slice=stream_slice, stream_state=stream_state
             )
             for record in parent_records:
-                yield {"repository": record["repository"], "project_id": str(record["id"])}
+                yield {"repository": record["repository"], "project_id": record["id"]}
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-github/source_github/streams.py:889]: Is there any reason we change str(..) to just as is it from record[...] ?

File: airbyte-integrations/connectors/source-github/source_github/streams.py [hunk 2]
```diff
@@ -925,3 +925,75 @@ def transform(self, record: MutableMapping[str, Any], stream_slice: Mapping[str,
...
+        for record in super().read_records(
+            sync_mode=sync_mode, cursor_field=cursor_field, stream_slice=stream_slice, stream_state=stream_state
+        ):
+            if record[self.cursor_field] > starting_point:
+                yield record
+
+    def get_starting_point(self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, Any]) -> str:
+        if stream_state:
+            repository = stream_slice["repository"]
+            project_id = str(stream_slice["project_id"])
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-github/source_github/streams.py:975]: Why do we coerce `str(stream_slice["project_id"])` to string here?

**Review by bazarnov — APPROVED** [review]
LGTM. Please check the comments about coercion to str().

File: airbyte-integrations/connectors/source-github/source_github/streams.py (hunk 1 above)
Comment (grubberr) [reply airbyte-integrations/connectors/source-github/source_github/streams.py:889]: we need this "project_id" in 2 places:
1) we put it inside state object (as key)
2) we put it inside record object (as value)

for (1) It need to be "string" type
for (2)  I preserve original "integer" type

**Review by sherifnada — APPROVED** [review]
LGTM but we should make a ticket to use caching for nested streams. Not blocking for beta.

**Pull request merged**